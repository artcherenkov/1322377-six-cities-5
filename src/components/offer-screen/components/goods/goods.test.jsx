import React from 'react';
import renderer from 'react-test-renderer';

import Goods from './goods';

const goods = [
  `Laptop friendly workspace`,
  `Washer`,
  `Towels`,
  `Fridge`,
  `Air conditioning`,
  `Baby seat`,
  `Breakfast`,
];

it(`Goods is rendered correctly`, () => {
  const tree = renderer.create(
      <Goods goods={goods} />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
