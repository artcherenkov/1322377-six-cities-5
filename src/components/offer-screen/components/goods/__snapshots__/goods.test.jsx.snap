// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Goods is rendered correctly 1`] = `
<div
  className="property__inside"
>
  <h2
    className="property__inside-title"
  >
    What's inside
  </h2>
  <ul
    className="property__inside-list"
  >
    <li
      className="property__inside-item"
    >
      Laptop friendly workspace
    </li>
    <li
      className="property__inside-item"
    >
      Washer
    </li>
    <li
      className="property__inside-item"
    >
      Towels
    </li>
    <li
      className="property__inside-item"
    >
      Fridge
    </li>
    <li
      className="property__inside-item"
    >
      Air conditioning
    </li>
    <li
      className="property__inside-item"
    >
      Baby seat
    </li>
    <li
      className="property__inside-item"
    >
      Breakfast
    </li>
  </ul>
</div>
`;
